contracts/LoanManager.sol

```diff
@@ -139,6 +142,8 @@ contract LoanManager is Restricted, TokenReceiver {
         require(product.minCollateralRatio > 0, "not a margin type loan");
 
         loan.collateralAmount = loan.collateralAmount.add(msg.value);
+
+        emit LoanChanged(loanId, loan.borrower, loan.collateralAmount, loan.repaymentAmount, getCurrentRate());
```

Comment:
I would consider calling this `LoanCollateralChanged`.

Reply:
we plan to use this for other kind of loan changes too in the future, e.g. partial repayment, so i would stick with the more general name